src/validation.cpp

```diff
@@ -3072,7 +3072,7 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, const Co
             fHaveWitness = true;
         }
 
-        const CBlockIndex* pindexForkBuffer = pindexPrev->GetAncestor(nHeight - BIP102_FORK_BUFFER);
```

Comment:
This makes `fSegwitSeasoned` become true immediately upon Segwit reaching `THRESHOLD_ACTIVE`.

NACK